sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -27,8 +27,8 @@ protected LogsIngestionClient()
         // request or stage as multiple blocks.
         internal static int SingleUploadThreshold = 1000000; // 1 Mb in byte format
 
-        // If no concurrency count is provided, default to serial upload (one block at a time).
-        private int DefaultWorkerCount = 1;
+        // If no concurrency count is provided for a parallel upload, default to 5 threads.
+        private int DefaultParallelWorkerCount = 5;
```

Comment:
Curious why we chose 5? Is this an ask from the service team or have we run some perf benchmark to decide this number?

Reply:
5 is the standard number in .Net. Other libraries like Storage use the same value

Reply:
Got it, thanks.